encoders/nlp/TransformerTorchEncoder/gpu.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM continuumio/miniconda3 AS conda
```

Comment:
would this Dockerfile also work for non-gpu environments?

Reply:
yes it would - but there would be a lot of "bloat". The GPU image will end up being ~3GB due to CUDA/torch, while i think the CPU should be about only 300MB